Sources/Fuzzilli/Mutators/ExplorationMutator.swift

```diff
@@ -46,12 +46,13 @@ public class ExplorationMutator: Mutator {
         case success = "Success"
         case cannotInstrument = "Cannot instrument input"
         case instrumentedProgramCrashed = "Instrumented program crashed"
+        case instrumentedProgramDifferential = "Instrumented program produced a differential"
```

Comment:
This is a good point; I had one instance producing non-reproducible differentials and this might be the root cause. In the next version of the PR I'll include the required functionality.

Reply:
The same is now true for the ProbingMutator :)